testing_and_setup/compass/ocean/drying_slope/marsh_flooding/idealized_transect/config_analysis.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<config case="analysis">
+  <add_link source="../comparison.py" dest="comparison.py"/>
```

Comment:
@caozd999, there should be a `base_path` specified here for when cases are setup in different working directories

Reply:
I believe it should be:
`<add_link path_base="script_test_dir" source="comparison.py" dest="comparison.py"/>`

Reply:
Thanks @sbrus89 , I have modified it.